### ISSUE TITLE
Polypen vertex selection area is too small

### ISSUE THREAD

**Issue by michelchafouin** [CGCookie/retopoflow#330]
Selecting a vertex in the polypen mode requires to be extra accurate, I need to zoom super close or click exactly on the pixels of the vertices (almost impossible).

Maybe it should use the Hotspot value set in the User Preferences > Interface.

**Comment by animatoring** [comment]
@michelchafouin What is the scale of the model you are working on? I noticed selection on smaller models can be an issue when selecting with polypen. Scaling the model up helped some.

**Comment by michelchafouin** [comment]
It's actually the opposite for me. If the mesh is big (25 meters sphere), I can't select anything.
If I try on a small sphere (1m sphere), it's better.

**Comment by animatoring** [comment]
Yeah I think it's a scale / clipping plane thing where only a certain size range is ideal and anything beyond that has the selection problem.

**Comment by michelchafouin** [comment]
Thanks for confirming the issue, I hope they will fix it :)

**Comment by patmo141** [comment]
I believe this is because picking/selection happens in 3d space, so thresholds are difficult to guess.  certainly we aim to have all of our tools behave, display, etc consistenly across all scales, orientations and transforms.  Difficult!!

**Comment by Herosnack** [comment]
TRUE, I have been working on a huge model (1000 BU), and retopoflow selection was too frustrating to work with. I had to resize to 5 BU to get it to work perfectly.
